### ISSUE TITLE
Discrepancy is the number of images in the zip vs. discrepancy

### ISSUE THREAD

**Issue by twosatnams** [switchablenorms/DeepFashion2#6]
Just download and unzipped the dataset zip files. The training set has 191961 images and the validation set has 32153. The description in the README.md says `The dataset is split into a training set (391K images), a validation set (34k images), and a test set (67k images).`

Is there going to be a second release with the remainder of the images?

**Comment by geyuying** [comment]
At present, half of the training set and full validation and test set is released for DeepFashion2 Challenge.

**Comment by zhouxiaoxu** [comment]
> At present, half of the training set and full validation and test set is released for DeepFashion2 Challenge.

so actually  the validation set has 32153 images?   and how can we get the other halr train set
